terraform/templates/cluster-test/ct

```diff
@@ -5,7 +5,9 @@ set -e
 
 DOCKER_IMAGE=""
 WAIT_TO="45m"
-export CONTAINER="cluster-test-$RANDOM"
+CONTAINER="cluster-test-$RANDOM"
```

Comment:
I think there was some reason to export it here(afaik it was required for one of lines below), lets keep it?

Reply:
Fixed